src/game/client/components/menus.cpp

```diff
@@ -1915,7 +1915,9 @@ int CMenus::Render()
 		else if(m_Popup == POPUP_REMOVE_FRIEND)
 		{
 			pTitle = Localize("Remove friend");
-			pExtraText = Localize("Are you sure that you want to remove the player from your friends list?");
+			pExtraText = Localize(m_pDeleteFriend->m_FriendState == CContactInfo::CONTACT_PLAYER
```

Comment:
Could you move that check outside of the Localize() call? We have a script that grabs all localization strings `Localize("string")` and it won't find these otherwise.

Reply:
Done, thanks for the heads-up.